lib/src/find_resource.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Contains code related to asset path resolution only. Not meant to be exposed.
+// This code is a modified version from ffigen package's old commit.
```

Comment:
If it's copied code, it should be in lib/src/third_party/. Probably lib/src/third_party/ffigen/ to clarify where the code is from. And then this comment line can be removed.